firefox-ios/Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -636,6 +636,7 @@ extension TelemetryWrapper {
         case crashedLastLaunch = "crashed_last_launch"
         case cpuException = "cpu_exception"
         case hangException = "hang-exception"
+        case tabLossDetected = "tab-loss-detected"
```

Comment:
@OrlaM Wondering if we should include additional details in the payload 🤔 Not sure how easily that data is searchable on the dashboard

Reply:
It won't show up on the dashboard but we could surface it on looker if we need to. I can't think of anything particularly useful to add. Previous tab count might be semi-useful as a hint but probably not worth it. We can always add to it later.